### PR TITLE
feat: page link to all our release notes

### PR DESCRIPTION
**Pull request by bearsyankees**
needs ux but functional

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ const Join = suspended(() => import('./pages/Join'));
 const Fall23Release = suspended(() => import('./pages/releases/fall23.mdx'));
 const QuistRelease = suspended(() => import('./pages/releases/quist.mdx'));
 const Tutorial = suspended(() => import('./components/Tutorial'));
+const ReleaseNotes = suspended(() => import('./pages/ReleaseNotes'));
```
Comment (Josh-Cena) [inline frontend/src/App.tsx:47]: Move to `/pages/releases/index.tsx`

File: frontend/src/components/Navbar/MeDropdown.tsx [hunk 2]
```diff
@@ -110,6 +116,10 @@ function MeDropdown({ isExpanded, setIsExpanded }: Props) {
             >
               Feedback
             </DropdownItem>
+            {/* Release notes */}
+            <DropdownItem icon={FcNews} to="/releases">
+              Release Notes
+            </DropdownItem>
```
Comment (Josh-Cena) [inline frontend/src/components/Navbar/MeDropdown.tsx:122]: I think it's better to not put it in the navbar, but we should add a link in the "About" page header

Comment (Josh-Cena) [reply frontend/src/components/Navbar/MeDropdown.tsx:122]: https://release-pages.preview.coursetable.com/about next to "Join Us" and "About"

**Comment by Josh-Cena** [comment]
@bearsyankees Instead of manually fighting with linting, you can run `bun checks:fix` to fix most of them, and `bun checks` to see what's left.

**Comment by bearsyankees** [comment]
> bun checks:fix

Yeah i do that and nothing usually gets fixed but i have no idea how to fix these style css linting now

**Comment by Josh-Cena** [comment]
I am really curious why the fixes aren't getting applied—have you tried running bun install again, and maybe `rm -rf node_modules` first?

**Comment by neilsong** [comment]
Dark mode broken by the way

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**